openshiftcontrolplane/v1/types.go

```diff
@@ -253,7 +257,7 @@ type BuildDefaultsConfig struct {
 	ImageLabels []buildv1.ImageLabel `json:"imageLabels,omitempty"`
 
 	// nodeSelector is a selector which must be true for the build pod to fit on a node
-	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+	NodeSelector *metav1.LabelSelector `json:"nodeSelector,omitempty"`
```

Comment:
You sure you want this one to be a pointer?  It's not touched by a user, so I'll let you have it this way if you want, but it's not obvious to me why you would want to.

Reply:
This may have been an oversight - non-pointer should be OK.

Reply:
Also missed updating NodeSelector here: https://github.com/openshift/api/pull/113/files#diff-c8352960961d4fca76c619d0d312d4efR293. `LabelSelector` provides a richer set of options that we're exposing to the user.